records_mover/db/bigquery/bigquery_db_driver.py

```diff
@@ -24,40 +25,25 @@ def __init__(self,
         super().__init__(db)
         self._bigquery_loader = BigQueryLoader(db=self.db, url_resolver=url_resolver)
 
-    def load(self,
-             schema: str,
-             table: str,
-             load_plan: RecordsLoadPlan,
-             directory: RecordsDirectory) -> int:
-        """Loads the data from the RecordsDirectory instance named
-        'directory'.  Guarantees a manifest file named 'manifest' is
-        written to the target directory pointing to the target
-        records.
+    def loader(self) -> Union[LoaderFromFileobj, LoaderFromRecordsDirectory]:
+        return self._bigquery_loader
 
-        Returns number of rows loaded (if database provides that
-        info).
+    def loader_from_fileobj(self) -> LoaderFromFileobj:
+        return self._bigquery_loader
 
-        """
-        return self._bigquery_loader.load(schema=schema,
-                                          table=table,
-                                          load_plan=load_plan,
-                                          directory=directory)
-
-    def load_from_fileobj(self, schema: str, table: str,
-                          load_plan: RecordsLoadPlan, fileobj: IO[bytes]) -> Optional[int]:
-        return self._bigquery_loader.load_from_fileobj(schema=schema,
-                                                       table=table,
-                                                       load_plan=load_plan,
-                                                       fileobj=fileobj)
+    def loader_from_records_directory(self) -> LoaderFromRecordsDirectory:
+        return self._bigquery_loader
 
+    # TODO: These next two shouldn't be in this class
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker